tests/rptest/tests/datalake/datalake_verifier.py

```diff
@@ -367,3 +367,13 @@ def stop(self):
         finally:
             if self._consumer:
                 self._consumer.close()
+        assert self._max_queried_offsets == self._max_consumed_offsets, "Mismatch between maximum offsets in topic vs iceberg table"
```

Comment:
is this intentionally duplicated from L362?

Reply:
looks like bad merge conflict resolution, i'll remove it